Bottom: 558a38bda76abdfcd83cac13c428ffef29cace69
Top:    f2cb185df8813e93fb40294769b300b8dba6a144
Author: Peter Clifton <peter@clifton-electronics.co.uk>
Date:   2015-02-15 20:17:37 +0000

Refresh of attempt-to-fix-broken-polygon

---

diff --git a/src/polygon1.c b/src/polygon1.c
index 054e9bc..e720c98 100644
--- a/src/polygon1.c
+++ b/src/polygon1.c
@@ -1703,8 +1703,8 @@ Collect (char poly, jmp_buf * e, PLINE * a, POLYAREA ** contours, PLINE ** holes
   VNODE *cur;
   DIRECTION dir = UNINITIALISED;
 
-  cur = (&a->head); //    ->next->next->next->next->next; /* Breaks circ_seg_test9.pcb */
   cur = (&a->head)        ;
+//  cur = (&a->head); //    ->next->next->next->next->next; /* Breaks circ_seg_test9.pcb */
   do
     {
       // The following may be a nice speedup, but not sure if it is correct.
